### ISSUE TITLE
Add XCMS object as data to EDAM Ontology

### ISSUE THREAD

**Issue by hechth** [sneumann/xcms#679]
In order to annotate tools with metadata, suitable EDAM ontology terms are very valuable. I couldn't find any term which would describe the summarized experiment or XCMS experiment object - would it be possible to add a suitable term to the EDAM ontology? Thanks very much!

**Comment by jorainer** [comment]
good point - I'll have a look into [EDAM](https://edamontology.org/page) and bring it also to Bioc.

**Comment by jorainer** [comment]
Having a quick look I am wondering: is it possible to annotate multiple EDAM terms to a e.g. `SummarizedExperiment`? We will for examples have:

- feature metadata (information on measured entities; maybe [data_0842](https://edamontology.github.io/edam-browser/#data_0842) )
- sample annotation (could be [data_3113](https://edamontology.github.io/edam-browser/#data_3113))
- abundance estimates (seems such a generic term does not yet exist in EDAM).

**Comment by hechth** [comment]
No, there is no way how to have one edam term map to multiple others I think. The other option would be to describe XCMS by using all of these terms basically.

**Comment by sneumann** [comment]
The xcms:multipleTerms annotation is not a limitation of EDAM. bio.tools or Workflowhub or the Bioschemas-beefed-up xcms Vignette could (untested!) allow to annotate a list of EDAM terms . Yours, Steffen
